Frameworks/UIKit/UIWindow.mm

```diff
@@ -360,10 +319,20 @@ - (UIWindowLevel)windowLevel {
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
+ @Notes Multi-screen apps can be built using WinRT APIs.
 */
 - (void)setScreen:(UIScreen*)screen {
-    UNIMPLEMENTED();
+    UNIMPLEMENTED_WITH_MSG("Multi-screen apps can be built using WinRT APIs.");
```

Comment:
This message seems unclear: my first parse resulted in "they can? cool! why are you crashing here though"